### DISCUSSION TITLE
VRoidHub連携画面はもう一回表示させる方法

### DISCUSSION THREAD

**Discussion by wcl1207** [pixiv/vroid-sdk-developers#140] [Q&A] [answered]
スクショのVRoid連携画面はもう一回表示させる方法がありますでしょうか。
<img width="738" alt="スクリーンショット 2024-02-14 11 54 35" src="https://github.com/pixiv/vroid-sdk-developers/assets/32382938/37eb5628-d5a0-4ef7-b0c6-1cc5fbd79e40">

背景は、アプリからVRoidHub連携画面に遷移したところ、iPhone SEでは、スクリーンショットのようにスクロールしても、赤い確認ボタンがほとんど見えない状態です。このバグを再確認したいのですが、一度連携確認ボタンを押した後、ログアウトし、再度ログインすると、自動的にログインされています。スクリーンショットのような連携画面が表示されませんでした。

アプリを削除し、再インストールしても、変わりません。自動的にログインされています。

ご回答いただけると、助かります。

端末：iPhone SE3 1.1.0

**Comment by nyamadan** [comment, marked as answer]
連携画面はアプリを使用しているアカウントでログイン後、次のページより「解除」ボタンを押していただくことでアプリとの連携が解除され、再度連携画面を表示することができます。

https://hub.vroid.com/settings/authorized_apps

横画面で連携ボタンが押せない不具合についてはVRoid Hubのウェブページを修正することで対応できると思いますのでこちらで修正いたします。

**Comment by wcl1207** [comment]
早速のご返事、ありがとうございます！
再度連携画面を表示させる方法は承知いたしました。

>横画面で連携ボタンが押せない不具合についてはVRoid Hubのウェブページを修正することで対応できると思いますのでこちらで修正いたします。

すみませんが、もし可能であれば、修正予定を教えていただけると、非常に助かります。

**Reply by nyamadan** [reply]
担当者をアサインし、遅くとも来週中に修正をリリースできるように計画しています。

修正内容としては連携画面内部をスクロールできるようにすることによって、狭い画面でも連携ボタンを押せるように修正することを計画しています。

修正が完了しましたらこちらのスレッドでお知らせいたします。

**Reply by nyamadan** [reply]
@wcl1207 

こちら、修正を展開いたしました。次のURLより連携を解除し、ご確認ください。

https://hub.vroid.com/settings/authorized_apps

**Reply by wcl1207** [reply]
ありがとうございます！！確認させていただきます。

**Reply by wcl1207** [reply]
@nyamadan 
早速のご対応、ありがとうございました！
「連携する」ボタンを見えるようになったことは確認できました！！
